src/main/java/com/qa/CVManager/Interoprability/Rest/UserCVFileRestController.java

```diff
@@ -0,0 +1,91 @@
+package com.qa.CVManager.Interoprability.Rest;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Optional;
+
+import org.bson.BsonBinarySubType;
+import org.bson.types.Binary;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.qa.CVManager.Persistence.Domain.User;
+import com.qa.CVManager.Persistence.Respository.UserRepository;
+
+@RestController
+@RequestMapping("/api")
+public class UserCVFileRestController {
+
+	@Autowired
+	UserRepository userRepo;
+
+	@PostMapping("/cvupload/{id}")
+	public String singleFileUpload(@RequestParam("file") MultipartFile multipart, @PathVariable String id) {
+
+		try {
+			Optional<User> optUser = userRepo.findById(id);
+			User c = optUser.get();
```

Comment:
c is a bad name for a variable

Reply:
Also as it's optional but you're doing get on it that defeats the purpose of an optional as you don't check whether there's a value to get, or use some default in case there isn't one